nucypher/characters/lawful.py

```diff
@@ -918,30 +911,31 @@ def retrieve(self,
             # TODO Optimization: Block here (or maybe even later) until map is done being followed (instead of blocking above). #1114
             the_airing_of_grievances = []
 
-            remaining_work_orders, capsules_to_activate = self._filter_work_orders_and_capsules(
-                new_work_orders, capsules_to_activate, m)
+            remaining_work_orders, remaining_capsules = self._filter_work_orders_and_capsules(
+                new_work_orders, message_kits, m)
```

Comment:
Phew - this method is a beast (#2028)!   We don't need to address it in this PR of course.